netlify.toml

```
@@ -1,3 +1,28 @@
 [build]
   command = "npm run build"
-  functions = "./dist/functions"
\ No newline at end of file
+  functions = "./dist/functions"
+
+[[redirects]]
+  from = "/win"
+  to = "/.netlify/functions/win-install"
+
+  # Proxy / rewrite transparently without the `location` hop.
+  status = 200
+
+  # Always redirect, even if somehow some content exists at the root "from"
+  force = true
+
+  headers = {X-From = "Netlify"}
```

I know I had this in my example, but it's worth scrutinizing.  Do we actually get value from passing this header on the (outgoing, from Netlify's proxy) request that we're proxying?  Do we consider this "from" Netlify?  I mean it is Netlify doing the proxying at our request, but maybe we want to describe it as this root proxy?  Is there use in gathering that or leveraging that in our scripts?

Also worth noting that [`X-` prefixed headers are deprecated/discouraged](https://tools.ietf.org/html/rfc6648) (now).

If we think there's value in supplying a header on that at all on that request, would it be useful to apply something like this for our own scripts/logging:

```suggestion
  headers = {Apollo-Proxy-Rule = "from-root-config"}
```

If not, perhaps we should just scrap this header until we have a purpose for it.

```suggestion
```

---

Yeah, that makes sense! Honestly, I wasn't 100% sure why that header was even there, or what its use was, so I have noo problem dropping it 😂